README.md

```diff
@@ -105,13 +105,138 @@ The goal of adding helm configurations is to abstract out and simplify the creat
+
+The following sample values show overriding at the Deployment, Pod, and Container levels:
+
+```yml
+AWX:
+  enabled: true
+  # These configurations relate to awx pods created by the operator, *not* the controller pods themselves
+  spec:
+    security_context_settings:
+      runAsNonRoot: true
+      runAsUser: 1001
+      seccompProfile:
+        type: RuntimeDefault
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop: [ "ALL" ]
+
+operator-controller-containers:
+  # this will get merged into the operator controller deployment spec for
+  # the container named `kube-rbac-proxy` at `spec.template.spec.containers`
+  kube-rbac-proxy:
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 1001
+      seccompProfile:
+        type: RuntimeDefault
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop: [ "ALL" ]
+  # example: override the environment variables of the main controller container
+  # note that since `env` is a list, it will be an override instead of a merge
+  # so any env var change requires re-specifying the whole list
+  awx-manager:
+    env:
+    - name: ANSIBLE_GATHERING
+      value: explicit
+    - name: ANSIBLE_DEBUG_LOGS
+      value: "true" # default was "false"
+    - name: WATCH_NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+
+
+operator-controller:
+  spec:
+    # replicas is an example spec for the deployment
+    replicas: 2
+    # template drills down into the pod that will be manaeg
+    template:
+      spec:
+        # This is a pod-level override, so it can be applied here
+        # and not worry about container list semantics
+        securityContext:
+          fsGroup: 2000
+
+```
+
+
+> Currently, this field cannot be used to add additional containers to the operator controller deployment
+
+
+### AWX
+
+| Value | Description | Default |
+|---|---|---|
+| `AWX.enabled` | Enable this AWX resource configuration | `false` |
+| `AWX.name` | The name of the AWX resource and default prefix for other resources | `"awx"` |
+| `AWX.annotations` | add annotations to the AWX resource | `{}` |
+| `AWX.labels` | add labels to the AWX resource | `{}` |
+| `AWX.spec` | specs to directly configure the AWX resource | `{}` |
+| `AWX.postgres` | configurations for the external postgres secret | - |
+
 
 The `AWX.postgres` section simplifies the creation of the external postgres secret. If enabled, the configs provided will automatically be placed in a `postgres-config` secret and linked to the `AWX` resource. For proper secret management, the `AWX.postgres.password` value, and any other sensitive values, can be passed in at the command line rather than specified in code. Use the `--set` argument with `helm install`. Supplying the password this way is not recommended for production use, but may be helpful for initial PoC.
```

Comment:
```suggestion
The `AWX.postgres` section simplifies the creation of the external postgres secret. If enabled, the configs provided will automatically be placed in a `postgres-config` secret and linked to the `AWX` resource. For proper secret management, the `AWX.postgres.password` value, and any other sensitive values, can be passed in at the command line rather than specified in code. Use the `--set` argument with `helm install`. Supplying the password this way is not recommended for public-facing workloads, but may be helpful for initial PoC.
```